pyo3-build-config/src/impl_.rs

```diff
@@ -1303,7 +1440,7 @@ fn conda_env_interpreter(conda_prefix: &OsStr, windows: bool) -> PathBuf {
 
 fn get_env_interpreter() -> Option<PathBuf> {
     match (env_var("VIRTUAL_ENV"), env_var("CONDA_PREFIX")) {
-        // Use cfg rather can CARGO_TARGET_OS because this affects where files are located on the
+        // Use cfg rather can CARGO_CFG_TARGET_OS because this affects where files are located on the
```

Comment:
Typo: "rather ~~can~~than"

Reply:
Applied in https://github.com/PyO3/pyo3/commit/9b45a19161cd20fac1c83c25897aa9d213244a15